### ISSUE TITLE
vscode remote explorer requires /bin/bash installed in host PC

### ISSUE THREAD

**Issue by jasonswee** [JunJie-zhang-o/EliteScript-jbi#1]
Hi, I was trying to use your extension to edit JBI scripts. However, I face the issue that when I was remote accessing it, it prompt that the Host does not have bash to support ssh connection. 

Can I find out how did you remote access with only sh shell available?

**Comment by JunJie-zhang-o** [comment]
First of all,this extension is only avaliable for JBI programs that edit Elite ec-series Robots.Secondly, I don’t quite understand what you said about remote connection errors because the plugin only has the most basic editing functions.

Normally, we use this plug-in to edit the program, and then use other plug-ins to transfer files, I'm not sure if you are talking about this problem.
Thanks for your question！！！

**Comment by jasonswee** [comment]
I see yes I am sorry I am referring to plugins that performs transferring the files. However, as I understand vscode, it has a remote explorer ability to realtime edit the remote computers' files in its IDLE like this https://code.visualstudio.com/docs/remote/ssh

**Comment by JunJie-zhang-o** [comment]
Yes, I probably understand what you mean. You should be talking about remote-ssh ，it is a excelllent plug-in. Unfortunately, our ec series currently does not support this function, because the plug-in was not available in the early design, and there are some system files in the file system. 

But our new cs series may support it! ! !

**Comment by jasonswee** [comment]
I see thank you for prompt response! I believe this issue can be closed!
